.jshintrc

```diff
@@ -1,28 +1,68 @@
 {
-    "browser": true,
+    "bitwise": false,
```

Comment:
interesting. all of those look legitimate except for this one:

```{js}
   src/data-table.js
    106 |            bAllFunctions = bAllFunctions & (typeof f === 'function');
```
which i'll fix when i merge this for the next beta release. it's harmless, but one still shouldn't use bitwise for booleans.

then there's the annoying but apparently very fast [`n  | 0`](http://jsperf.com/javascript-trunc)...